test/timestamps.test.js

```diff
@@ -608,12 +608,13 @@ describe('timestamps', function() {
     });
 
     it('should change updatedAt when findOneAndUpdate', async function() {
+      await Cat.deleteMany({});
       await Cat.create({ name: 'test123' });
       let doc = await Cat.findOne({ name: 'test123' });
       const old = doc.updatedAt;
+      await new Promise(resolve => setTimeout(resolve, 10));
```

Comment:
what requires this sleep?

Reply:
This test flaked once while I was testing, so I figured I'd make this test more robust while it was fresh in my mind. This test doesn't always fail, but you can see that without the sleep this test can fail.